code/game/gamemodes/disaster/disaster.dm

```diff
@@ -0,0 +1,58 @@
+/datum/game_mode/disaster
+	name = "disaster"
+	config_tag = "disaster"
+	report_type = "disaster"
+	false_report_weight = 0
+	required_players = 0
+
+	announce_span = "danger"
+	announce_text = "Catastrophic disasters in-bound!"
+
+var/finished = 0
+
+/datum/game_mode/disaster/pre_setup()
+	SSevents.scheduled = 3000
```

Comment:
This should be `world.time + 5 MINUTES`

Reply:
the reason this didn't work out is because world.time ticks the entire time the server is open, not just when the round starts